src/edu/ucsb/nceas/metacat/restservice/v2/CNResourceHandler.java

```diff
@@ -135,7 +135,7 @@
     public CNResourceHandler(ServletContext servletContext,
             HttpServletRequest request, HttpServletResponse response) {
         super(servletContext, request, response);
-        logMetacat = Logger.getLogger(CNResourceHandler.class);
+        logMetacat = LogFactory.getLog(CNResourceHandler.class);
```

Comment:
Closing the streams here looks great!

Reply:
I don't why you saw the statement like this - `logMetacat = Logger.getLogger(CNResourceHandler.class);`. It has already changed to `logMetacat = LogFactory.getLog(CNResourceHandler.class);`